pkg/node/uevent.go

```diff
@@ -439,6 +439,23 @@ func StartDynamicDriveHandler(ctx context.Context, identity, nodeID, rack, zone,
 		dynamicDriveDiscovery: true,
 		loopbackOnly:          loopbackOnly,
 	}
+
+	return handler.processLoop(ctx)
+}
+
+func syncDrives(ctx context.Context, identity, nodeID, rack, zone, region string, loopbackOnly bool) error {
```

Comment:
sync drives should not need a handler. We are not hadling uevents for initial sync